modules/mri_violations/js/mri_protocol_check_violations.js

```diff
@@ -10,4 +10,29 @@ $(document).ready(function(){
 $(window).resize(function(){
     Table.checkOverflow("content", "scrollRight", "scrollLeft");
     // checkOverflow();
-});
\ No newline at end of file
+});
+$(window).ready(function () {
+       
+       var filter = getParameterByName('filter');
+     
+       if(filter==="true"){
+         
+            $( "input[name ='filter']" ).click();
+       
+        }
+ });
+
+function getParameterByName(name, url) {
```

Comment:
What's this function supposed to be doing? Based on the name, it seems to be just trying to get a URL parameter, but it seems to be doing it in a very complex way. There's already a built in helper function for doing it (talk to @alisterdev for details.)